shell/common/animator.cc

```diff
@@ -174,6 +174,9 @@ void Animator::Render(std::unique_ptr<flutter::LayerTree> layer_tree) {
                                 "Animator::Render");
   frame_timings_recorder_->RecordBuildEnd(fml::TimePoint::Now());
 
+  delegate_.OnAnimatorUpdateLatestFrameTargetTime(
```

Comment:
Is this safe to access without a lock?

Reply:
I think it's safe, the lock is in `Shell::OnAnimatorUpdateLatestFrameTargetTime`

Reply:
Sorry, I meant for this comment to be in the line below this (`frame_timings_recorder_->GetVsyncTargetTime()`

Reply:
I see now though that takes a lock by itself.